server/src/db/queries/pathway/createUpdatePathway.js

```diff
@@ -2,54 +2,53 @@ const Query = require('../../core/Query')
 const Model = require('../../core/Model')
 
 const createUpdatePathway = ({id, name, steps}) => {
-    // create id for path if doesn't exist
-    if (!id) {
```

Comment:
This is unnecessary is ID is a mandatory field right?

Reply:
Yeah id is mandatory. That's why I've removed it. The query will not reach here without and id value.

Reply:
Sorry, my bad